crates/aptos-telemetry/src/metrics.rs

```diff
@@ -5,6 +5,25 @@ use aptos_metrics_core::{register_int_counter_vec, IntCounterVec};
 use once_cell::sync::Lazy;
 
 /// Counter for successful telemetry events
+pub(crate) static APTOS_TELEMETRY_SERVICE_SUCCESS: Lazy<IntCounterVec> = Lazy::new(|| {
```

Comment:
What's the difference between this and `APTOS_TELEMETRY_SUCCESS` ?

Reply:
The naming might not be the best, but since I am keeping the old logic to post metrics to Google Analytics, I am creating new counters for the telemetry service instead of using the existing ones.

Reply:
Let's add a comment here for clarity and also I am assuming the old counters will be deprecated soon, can you add a TODO as well?